ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/switchport-default.j2

```diff
@@ -5,6 +5,9 @@
 ### Switchport Defaults Summary
 
 - Default Switchport Mode: {{ switchport_default.mode }}
+- Default Switchport Phone COS: {{ switchport_default.phone.cos }}
```

Comment:
you need to add protection when these vars are not defined: `if variable is arista.avd.defined`